solutions/c/02-cz2/explanation.md

```diff
@@ -0,0 +1,17 @@
+The entry point for your Shell implementation is in `app/main.c`.
```

Comment:
@andy1li this is wrong - we seem to be wrongly compiling explanations for stage 2 from stage 1. We need to keep these empty / not generate the file

Reply:
Got it! Let me check how to do that.